content/guides/dashboard/organizations.md

```diff
@@ -114,11 +114,26 @@ If you have any questions regarding the OSS plan, please feel free
 
 ## Integrations
 
+### BitBucket Integration
+
+Please see our full documentation for
+[BitBucket Integration](/guides/dashboard/bitbucket-integration).
+
 ### GitHub Integration
 
 Please see our full documentation for
 [GitHub Integration](/guides/dashboard/github-integration).
 
+### GitLab Integration
+
+Please see our full documentation for
+[GitLab Integration](/guides/dashboard/gitlab-integration).
+
+### Jira Integration
+
+Please see our full documentation for
+[Jira Integration](/guides/dashboard/jira-integration).
+
 ### Slack Integration
 
 Please see our full documentation for
```

Comment:
@conversaShawn Instead of having a section for each integration we can just have one bulleted list of the integration, under this "Integrations" section.

Reply:
@amirrustam  Great point. Changes made.